src/components/cart/proceed-payment/index.tsx

```diff
@@ -0,0 +1,242 @@
+              name={data.cartData.textboxLabel}
+              isPassword={false}
+              style={{
+                width: "263px",
+                margin: "auto",
+                color: `${colors.grey_4a4a4a}`,
+              }}
+            />
+          </StepsSection>
+          <PaymentContainer>
+            <RadioTitle>{data.offersRadio.title}</RadioTitle>
+            <Wrapper>
+              <RadioInput
+                type="radio"
+                id="userCode"
+                name="referal"
+                value="userCode"
+                onClick={handleRadioClick}
+                checked={radioButtons.userCode}
+              />
+              <OptionName htmlFor="userCode">
+                {data.offersRadio.options[0]}
+              </OptionName>
+            </Wrapper>
+            {radioButtons.userCode === true ? (
+              <UserCodeSection>
+                <InputField
+                  name="Add your promo code"
+                  isPassword={false}
+                  style={{ width: "238px", height: "33px" }}
+                />
+                <AlertText content={data.offersRadio.alertsText} />
+              </UserCodeSection>
+            ) : null}
+            <Wrapper style={{ marginTop: "24px" }}>
+              <RadioInput
+                type="radio"
+                id="bestOffer"
+                name="referal"
+                value="bestOffers"
+                onClick={handleRadioClick}
+                checked={radioButtons.bestOffers}
+              />
+              <OptionName htmlFor="best-offer">
+                {data.offersRadio.options[1]}
+              </OptionName>
+            </Wrapper>
+          </PaymentContainer>
+        </CartSection>
+      </ContentContainer>
+    </PageSection>
+  );
+};
+export default ProceedPayment;
+
+const PageSection = styled.div`
+  width: 100wh;
+  height: 842px;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  justify-content: space-between;
+  gap: 40px;
+  padding-bottom: 72px;
+`;
+
+const ContentContainer = styled.div`
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  justify-content: space-between;
+  gap: 39px;
+  margin-left: auto;
+  margin-right: auto;
+`;
+
+const ProgressSection = styled.div`
+  height: 55px;
+  width: 66%;
+`;
+
+const CookingInstructionTitle = styled.div`
+  height: 17px;
+  width: 122.95px;
+  color: ${colors.grey_858585};
+  font-family: ${fontFamilies.fontFamilyOsRegular};
+  font-size: ${sizes.size12};
+  letter-spacing: ${sizes.sizeNeg0_24};
+  line-height: ${sizes.size17};
+  margin-left: 21px;
+  margin-top: 17px;
+  margin-bottom: 15px;
+`;
+
+const CartSection = styled.div`
+  width: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 23px;
+  box-sizing: border-box;
+`;
+
+const StepsSection = styled.div`
+  width: 634px;
+`;
+
+const StepTitle = styled(Label)`
```

Comment:
StepTitleLabel

Reply:
StepTitle changed to StepTitleLabel, will be adding label in future usages also.